<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adjustments</class>
 <widget class="QDialog" name="Adjustments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Pb_Down">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>\/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pb_Right">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pb_Up">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>/\</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pb_Left">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Pb_Ok">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Sp_Delay">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay between steps (&gt;=1)</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="Sp_RotationDegrees">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-360</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Degrees to rotate&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Pb_Rotate">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
